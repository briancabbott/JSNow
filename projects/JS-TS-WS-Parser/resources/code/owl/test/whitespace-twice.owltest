---
.whitespace 'x'
.whitespace 'x'
a = 'a'
