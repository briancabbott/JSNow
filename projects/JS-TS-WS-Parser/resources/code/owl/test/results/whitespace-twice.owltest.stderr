error: this whitespace was specified twice

  .whitespace 'x'
              ~~~
  .whitespace 'x'
              ~~~

